### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by KulbhushanSaxenaTR**

### CODE REVIEW

File: src/main/java/week2/LeftRotation.java [hunk 1]
```diff
@@ -0,0 +1,44 @@
+package week2;
+import java.util.Arrays;
+import java.util.Scanner;
+
+public class LeftRotation {
+
+    private static void rotate(int[] a, int d) {
+        while(d>0) {
+            int start = a[0];
+            for(int i=0; i<a.length-1; i++) {
```
Comment (Misto1997) [inline src/main/java/week2/LeftRotation.java:12]: Try to improve it to O(n).
explore discussion section or editorial for some exiting  and better solution

File: src/main/java/week2/NewYearChaos.java [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    q[i-1] = q[i];
+                    q[i] = temp; 
+                    bribe[q[i]]++;
+                    count++;
+                    if(bribe[q[i]] > 2) {
+                        System.out.println("Too chaotic");
+                        check++;
+                        break;
+                    }
+                    isSorted = false;
```
Comment (Misto1997) [inline src/main/java/week2/NewYearChaos.java:27]: Code modularity is still missing on every problem solution.

File: src/main/java/week2/LeftRotation.java (hunk 1 above)
Comment (KulbhushanSaxenaTR) [reply src/main/java/week2/LeftRotation.java:12]: Ok Will try to see some better solution from discussions

File: src/main/java/week2/NewYearChaos.java (hunk 2 above)
Comment (KulbhushanSaxenaTR) [reply src/main/java/week2/NewYearChaos.java:27]: Sure will keep in mind

**Comment by KulbhushanSaxenaTR** [comment]
Added problem links to the java files for week1 and week2

**Review by Misto1997 — APPROVED** [review]

**Pull request merged**